src/shared/components/palette/document/read-document-field.component.spec.ts

```diff
@@ -72,7 +89,19 @@ describe('ReadDocumentFieldComponent', () => {
       expect(text(de)).toEqual(VALUE.document_filename.toString());
       let linkElement = de.query(By.css('a'));
       expect(linkElement).toBeTruthy();
-      expect(attr(linkElement, 'href')).toEqual(GATEWAY_DOCUMENT_URL);
+      expect(attr(linkElement, 'href')).toEqual(DOCUMENT_CLICKABLE_HREF);
+    });
+
+    it('should call Media Viewer when the document link is clicked', () => {
+      component.caseField.value = VALUE;
+      fixture.detectChanges();
+      spyOn(component, 'showMediaViewer');
+      let linkElement = de.query(By.css('a'));
+      expect(linkElement).toBeTruthy();
+      linkElement.triggerEventHandler('click', null);
+      fixture.detectChanges();
+      component.showMediaViewer();
```

Comment:
Should we not remove this line and assert based on the click on a link only?

Reply:
Removed.